README.md

```diff
@@ -25,6 +27,10 @@ config :pool_max, :validate => :number, :default => 50
 # Max number of concurrent connections to a single host. Defaults to 25
 config :pool_max_per_route, :validate => :number, :default => 25
 
+# Specify this if you want to keep the TCP connection open between requests
+# using HTTP Keep-Alive. Defaults to 0 (off)
+config :keepalive, :validate => :number, :default => 0
```

Comment:
Awesome, thanks for that research! 
@andrewvc - let's make keepalive `:validate => :boolean`.